equinox/nn/pool.py

```diff
@@ -27,6 +28,7 @@ def __init__(
         kernel_size: Union[int, Sequence[int]],
         stride: Union[int, Sequence[int]] = 1,
         padding: Union[int, Sequence[int], Sequence[Tuple[int, int]]] = 0,
+        use_ceil: bool = False,
```

Comment:
I think for readability's sake this argument needs to be propagated through to all of `MaxPool2d` etc., so that it's in their signature and docstring. (At the moment it's just being passed through as a `**kwarg`.)